src/styles/blocks/nav.scss

```
@@ -0,0 +1,55 @@
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    margin-left: 20px;
+    display: flex;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-size: 12px;
+    font-weight: $font-w-s;
+    line-height: 14px;
+    text-transform: uppercase;
+    color: $card-text-color;
+    text-decoration: none;
+    white-space: nowrap;
+
+    transition: color 0.3s;
+
+    &:hover {
+      color: $main-blue-color;
+    }
+
+    &--is-active {
+      color: $main-blue-color;
+    }
+
+    &--is-active::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      width: 37px;
```

```suggestion
      width: 100%;
```
try to add is-active for another link of header
it will not work if **width: 37px;**